result/bgpv4.yaml

```diff
@@ -24,35 +24,89 @@ components:
           description: >-
             The list of column names that the returned result set will contain.
             If the list is empty then all columns will be returned except for
-            any result_groups.
-            The name of the BGPv4 device cannot be excluded.
+            any result_groups. The name of the BGPv4 peer cannot be excluded.
           type: array
           items:
             type: string
             enum:
               - session_state
-              - routes_advertised
-              - routes_withdrawn
+              - flap_count
+              - routes_sent
+              - routes_received
+              - route_withdraws_sent
+              - route_withdraws_received    
+              - updates_sent
+              - updates_received
+              - opens_sent
+              - opens_received
+              - keepalives_sent
+              - keepalives_received
+              - notifications_sent
+              - notifications_received
 
     Bgpv4.Metric:
       description: >-
-        BGPv4 Router statistics and learned routing information
+        BGPv4 per peer statistics information.
       type: object
       properties:
         name:
           description: >-
-            The name of a configured BGPv4 device.
+            The name of a configured BGPv4 peer.
           type: string
         session_state:
           description: >-
-            Session state as up or down
+            Session state as up or down.
```

Comment:
Since we're on it, I think it would be useful to provide more details on what `up` or `down` stands for here.

i.e. my understanding is `up` refers to `Established` state, and `down` refers to others. we of course, might need to rephrase a little more.

Reply:
yes. we can